providers/aws/resources/aws_acm.go

```diff
@@ -69,13 +69,20 @@ func (a *mqlAwsAcm) getCertificates(conn *connection.AwsConnection) []*jobpool.J
 					return nil, err
 				}
 				for _, cert := range certs.CertificateSummaryList {
-					mqlCert, err := CreateResource(a.MqlRuntime, "aws.acm.certificate",
-						map[string]*llx.RawData{
-							"arn": llx.StringData(convert.ToString(cert.CertificateArn)),
-						})
+					// ListCertificates does not return the subject and tags, so we need to call DescribeCertificate
+					// and ListTagsForCertificate for each certificate, which initAwsAcmCertificate does.
```

Comment:
i think we're supposed to use NewResource here?  and then init gets automatically called to fill things in

Reply:
ok, thanks. I pushed a fix